content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -115,24 +82,85 @@ The Istio control plane must be installed with virtual machine integration enabl
     $ istioctl install -f "${WORK_DIR}"/vmintegration.yaml
     {{< /text >}}
 
-## Create files to transfer to the virtual machine
+## Create Namespace and Setup policies for VM
+1. Create Namespace that will host the VM
+
+    {{< text bash >}}
+    $ kubectl create namespace $SERVICE_NAMESPACE
+    {{< /text >}}
+
+1. Create a serviceaccount for VM
+
+    {{< text bash >}}
+    $kubectl create serviceaccount $SERVICE_ACCOUNT -n $SERVICE_NAMESPACE 
+    {{< /text >}}
+
+1. Get the external ip or internal ip of the VM and store to `"${VM_IP}"`
```

Comment:
this step needs more explination without going into platform specfics.

Reply:
add explanation like for example nat external ip 34.94.87.55 in Google Cloud or IPv4 Public IP    18.183.135.37 in AWS